openfisca_us/parameters/gov/states/mo/tax/income/rates.yaml

```
@@ -0,0 +1,68 @@
+description: The tax rates for individual residents in Missouri. Thresholds are <=.
+brackets:
+- rate:
+    2021-01-01:
+      value: 0.015  
+  threshold:
+    2021-01-10:
+      value: 0
+- rate:
+    2021-01-10:
+      value: 0.02  
+  threshold:
+    2021-01-10:
+      value: 1000
```

Please include thousands separators e.g.
```suggestion
      value: 1_000
```